<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="teOutput"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>141</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWeb_API">
    <property name="title">
     <string>Web API</string>
    </property>
    <addaction name="actionGrant"/>
    <addaction name="actionGet_Playlists"/>
    <addaction name="actionGet_User_Information"/>
   </widget>
   <addaction name="menuWeb_API"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGrant">
   <property name="text">
    <string>Grant</string>
   </property>
  </action>
  <action name="actionGet_Playlists">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get Playlists</string>
   </property>
   <property name="toolTip">
    <string>Query Playlists from Webapi</string>
   </property>
  </action>
  <action name="actionGet_User_Information">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get User Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
